cypress/integration/advanced-functionalities/sidebar/compose/Manage_namespaces.spec.js

```diff
@@ -0,0 +1,30 @@
+/// <reference types="cypress" />
+const composeURL = Cypress.env('COMPOSE_URL')
+const email = Cypress.env('USER_EMAIL')
+const password = Cypress.env('USER_PASSWORD')
+
+describe('Test for compose sidebar manage namespaces button functionality', () => {
+  before(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ email, password, url: composeURL })
+    }
+  })
+
+  context('Test for compose sidebar manage namespaces button functionality', () => {
+    it('should be able to click on manage namespaces and be redirected', () => {
+      cy.visit(composeURL + '/namespaces')
+      // We wait for 3s in order the page to be fully loaded
+      cy.wait(3000)
+      cy.get('[data-test-id="input-search"]').type('crm')
+      // We wait for 1s in order the search to be finished
+      cy.wait(1000)
+      // We need to visit a namespace so that the sidebar will be present
+      cy.get('[data-test-id="link-visit-namespace"]').click({ force: true })
```

Comment:
I've appended the namespace short name to the cypress id. Please, apply it here.